sqlmesh/core/renderer.py

```diff
@@ -173,7 +174,7 @@ def _resolve_table(table: str | exp.Table) -> str:
         )
 
         start_time, end_time = (
-            make_inclusive(start or c.EPOCH, end or c.EPOCH, self._dialect)
+            make_inclusive(start or c.EPOCH, end or c.EPOCH + timedelta(days=1), self._dialect)
```

Comment:
I don't think this is the only place where we use epoch as a placeholder. I wonder whether we should instead have 3 constants:
```
START_PLACEHOLDER
END_PLACEHOLDER
EXECUTION_TIME_PLACEHOLDER
```
and use these instead throughout.